gazebo/test/ServerFixture.hh

```diff
@@ -267,7 +268,9 @@ namespace gazebo
                    bool _distortion = false, double _distortionK1 = 0.0,
                    double _distortionK2 = 0.0, double _distortionK3 = 0.0,
                    double _distortionP1 = 0.0, double _distortionP2 = 0.0,
-                   double _cx = 0.5, double _cy = 0.5);
+                   double _cx = 0.5, double _cy = 0.5,
+                   bool _legacyMode = true,
+                   double _horizontalFov = 0.78539816339744828);
```

Comment:
> I think this breaks ABI?
> 
> ABI checker didn't show any issue here but I'm not sure if that's because it's not testing the `ServerFixture` library.

I was wondering the same thing, but we only distribute `ServerFixture` in a static library, and I believe ABI is only a concern for dynamically loaded libraries